package.json

```diff
@@ -41,36 +41,35 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^27.0.6",
-    "babel-loader": "^8.0.5",
+    "babel-loader": "^8",
     "cross-fetch": "^3.0.6",
-    "css-loader": "^2.1.0",
+    "css-loader": "^6",
     "cypress": "^3.1.4",
     "eslint": "^7",
     "eslint-config-mavenlint-react": "^2.0.0",
     "eslint-plugin-cypress": "^2.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^27.0.6",
     "msw": "^0.26.0",
-    "postcss-loader": "^4.0.4",
+    "postcss-loader": "^6",
     "postcss-preset-env": "^6.7.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-ga": "^2.5.6",
-    "react-styleguidist": "^11.1.7",
+    "react-styleguidist": "juanca/react-styleguidist#master",
```

Comment:
Nice! How much work did this take? 😅

Reply:
Easy peasy! But it is using an unreleased version of one of its dependencies. But people with context are saying its close. So I'll keep an eye. Hopefully all of it gets released before we finish the Webpack 5 migration in bigmaven.